spec/feature/user/edits_a_listing_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+describe 'user edits a listing', type: :feature do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.